lib/SoyOptions.js

```diff
@@ -21,6 +21,18 @@ function SoyOptions() {
   this.inputDir = process.cwd()
 
   /**
+   * Directory containing the Closure Templates SoyToJsSrcCompiler.jar
+   * @type {string}
+   */
+  this.soyJarDir = process.cwd()
```

Comment:
If passed in as an argument, it gets checked dynamically at:

https://github.com/gawkermedia/soynode/pull/1/files#diff-77bf383a42fff8dd7d74b595e9c73d95L180

And then on consumer side, the path is defined like this (though hopefully without need for the hardcoded node_modules path):
https://github.com/gawkermedia/kinja-mantle/commit/f47e6e6d0c840d26c8dcd5dd1715a1a07f84695a#diff-f0807c322f54cb38e739cc904149f38bR34